centreon/src/Core/ServiceTemplate/Infrastructure/Configuration/symfony.yaml

```
@@ -0,0 +1,25 @@
+
+  Core\ServiceTemplate\Application\Repository\WriteServiceTemplateRepositoryInterface:
+    class: Core\ServiceTemplate\Infrastructure\Repository\DbWriteServiceTemplateRepository
+
+  Core\ServiceTemplate\Infrastructure\Repository\DbWriteServiceTemplateActionLogRepository:
+    decorates: Core\ServiceTemplate\Application\Repository\WriteServiceTemplateRepositoryInterface
+    arguments:
+      - '@.inner'
+      - '@Centreon\Domain\Contact\Interfaces\ContactInterface'
+      - '@Core\ServiceTemplate\Application\Repository\ReadServiceTemplateRepositoryInterface'
+      - '@Core\ActionLog\Application\Repository\WriteActionLogRepositoryInterface'
+      - '@Centreon\Infrastructure\DatabaseConnection'
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)